client/state/stats/lists/selectors.js

```diff
@@ -204,6 +204,14 @@ export function getSiteStatsViewSummary( state, siteId ) {
 		viewSummary[ years ][ months ].average = round( average, 0 );
 	} );
 
+	// With the current month, calculate the average based on the days passed so far in the month
+	const thisMonth = viewSummary[ moment().year() ][ moment().month() ];
+	if ( thisMonth ) {
+		const daysSinceStartOfMonth =
+			moment( new Date() ).diff( moment().startOf( 'month' ), 'days' ) + 1;
```

Comment:
In this first `moment()` call, can't we use the date we just used above for retrieving the current month?